packages/components/pie-button/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+@customElement('pie-button')
```

Comment:
Are we omitting semicolons intentionally?

Reply:
Good question but I don't think you're supposed to use them on the end of decorators